build-system/tasks/helpers.js

```diff
@@ -380,7 +380,7 @@ function finishBundle(srcFilename, destDir, destFilename, options) {
  * @param {?Object} options
  * @return {!Promise}
  */
-async function compileUnminifiedJs(srcDir, srcFilename, destDir, options) {
+async function esbuildJs(srcDir, srcFilename, destDir, options) {
```

Comment:
Is there a reason this name needs to change with this PR?

We are still compiling the unminified js with this method right?

Reply:
It was compiling "unminified" core code and npm code (ESM and ES5), so the name change seemed appropriate.